extensions/two-factor-authentication-code-generator/package.json

```
@@ -5,6 +5,9 @@
   "description": "Quickly access your two-factor authentication codes.",
   "icon": "command-icon.png",
   "author": "cjdenio",
+  "contributors": [
+    "lunareclipsewasshort"
```

```suggestion
    "lunareclipsewasshort",
    "litomore"
```

Would you mind adding me to the contributors? I want to keep an eye on any changes on this extension. To ensure that future updates do not make breaking changes.